force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.html

```diff
@@ -131,7 +131,7 @@
                                         {labels.every}
                                     </lightning-layout-item>
                                     <lightning-layout-item
-                                        size="3"
+                                        size="4"
```

Comment:
NOTE: The width of the modal is much narrower in experience cloud, so I made this slightly bigger so the field is usable.